install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -6,57 +6,107 @@ schema: |-
   definition user {}
 
   definition organization {
+    // Organization specific roles
     relation owner: user
     relation member: user
 
-    permission organization_read = owner + member
-    permission organization_write = owner
+    // Membership is the union of all organization specific roles
+    permission membership = owner + member
 
-    permission organization_members_read = owner + member
-    permission organization_members_write = owner + member
+    // General operations on organization
+    permission read_info = owner + member
+    permission write_info = owner
+
+    // Operations on Organization's Members
+    permission read_members = owner + member
+    permission write_members = owner + member
```

Comment:
Sorry for being late here, but: Was this `+ member` intended? :thinking:

Reply:
We can change it, but currently any member of an org can invite others

Reply:
Ah, my bad: Everbody can create an invite link :+1: